apps/home/routes.py

```diff
@@ -766,10 +766,34 @@ def add_answer_sheet():
 
     return render_template("pages/lab_answers_sheet.html", labs=labs, lab_id=lab_id, answers=answers, msg_ok="Lab answer sheet saved!")
 
+@blueprint.route('/feedback/hide', methods=["POST"])
+@login_required
+def hide_feedback():
+    data = request.get_json() if request.is_json else request.form
+    feedback_id = data.get("feedback_id")
+    action = data.get("action")
+    if not feedback_id or action not in ["hide", "unhide"]:
+        return {"status": "fail", "result": "Missing feedback_id or invalid action"}, 400
```

Comment:
Instead of returning pure JSON, it would  be nice if you return to the route with a message to the feedbacks page that can open a Toasts/message to display the results to the requestor.